private/queue_private.h

```diff
@@ -493,7 +493,7 @@ dispatch_async_enforce_qos_class_f(dispatch_queue_t queue,
  * "detached" before the thread exits or the application will crash.
  */
 DISPATCH_EXPORT
-void _dispatch_install_thread_detach_callback(void (*cb)(void));
+void _dispatch_install_thread_detach_callback(dispatch_function_t cb);
```

Comment:
No, this should remain as is.  The parameter is never passed to the callback.  @adierking fixed this.

Reply:
I’m not sure I follow, but the [current implementation](https://github.com/apple/swift-corelibs-libdispatch/blob/13129d9/src/queue.c#L7306) of this method uses the signature proposed in this patch. Should I change the implementation signature instead?

Reply:
I changed this from `dispatch_function_t` in #403. It seems that the change to queue.c may have been reverted on accident. Let's fix it there.